theforeman.org/scripts/test/test_katello.sh

```diff
@@ -1,79 +0,0 @@
-#!/bin/bash -ex
```

Comment:
This file is still used - probably in the source build step for the nightly package.

Reply:
It's used by plugin testing for foreman-virt-who-configure, I meant to not trash it yet.